src/main/java/uk/gov/pay/ledger/filters/LoggingMDCResponseFilter.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.pay.ledger.filters;
+
+import org.slf4j.MDC;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import java.io.IOException;
+
+public class LoggingMDCResponseFilter implements ContainerResponseFilter {
+
+    @Override
+    public void filter(ContainerRequestContext containerRequestContext, ContainerResponseContext containerResponseContext) throws IOException {
+        MDC.clear();
```

Comment:
What's the reason why in some of the apps we specify which MDC keys to remove? I'm not familiar with this stuff.

Reply:
I'm not sure either - in publicauth the only key logged there is account_id so we can just specify to remove that. I'm pretty sure we have to clear all of the MDC keys as its thread local.

Reply:
yeah we could have probably used MDC.clear everywhere.